trailmet/models/mobilenet.py

```diff
@@ -13,10 +13,10 @@
 '''
 
 
-class Block(nn.Module):
+class InvertedResidual(nn.Module):
```

Comment:
@Arnav0400 Is it okay to keep `InvertedResidual` name itself?

Reply:
should be fine